lib/cache.js

```diff
@@ -1,8 +1,31 @@
 'use strict'
 
+const Winston = require('winston')
 const LevelUp = require('level')
+const Sublevel = require('level-sublevel')
 const Path = require('path')
 
-const path = Path.join(__dirname, '..', 'data')
+const path = Path.join(__dirname, '..', 'cache')
```

Comment:
The cleanup is done on the db reference and not on the directory itself, this was the only place where the directory was specified